buildSrc/src/main/java/Dependencies.kt

```diff
@@ -32,6 +32,8 @@ object Versions {
 object Dependencies {
     const val gradle_build_tool = "com.android.tools.build:gradle:${Versions.gradleBuildTool}"
     const val databinding = "androidx.databinding:compiler:${Versions.dataBinding}"
+    const val  appcompat= "androidx.appcompat:appcompat:1.0.0-beta01"
```

Comment:
```suggestion
    const val appcompat = "androidx.appcompat:appcompat:1.0.0-beta01"
```

this part - `1.0.0-beta01` should be in a variable above in this file, just to keep consistency :)